htdocs/install/mysql/tables/llx_c_productbatch_status.key.sql

```diff
@@ -0,0 +1,19 @@
+-- ========================================================================
+-- Copyright (C) 2021 Noé Cendrier <noe.cendrier@altairis.fr>
+--
+-- This program is free software; you can redistribute it and/or modify
+-- it under the terms of the GNU General Public License as published by
+-- the Free Software Foundation; either version 3 of the License, or
+-- (at your option) any later version.
+--
+-- This program is distributed in the hope that it will be useful,
+-- but WITHOUT ANY WARRANTY; without even the implied warranty of
+-- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+-- GNU General Public License for more details.
+--
+-- You should have received a copy of the GNU General Public License
+-- along with this program. If not, see <https://www.gnu.org/licenses/>.
+--
+-- ========================================================================
+
+ALTER TABLE llx_c_productbatch_status ADD UNIQUE INDEX uk_c_productbatch_status(code);
```

Comment:
Can you add column "entity" into the unique key  (code, entity)

Reply:
I certainly could, but I’m not sure I understand the purpose of it. In my understanding, the _usability status_, which is what this really is, is not directly related to the entity. I have not, at this point, an entity field in the table, and it feels, if I had to had one, that it should more a foreign key than a unique key. So please tell me more on this.

Reply:
Or, maybe (and it would make perfect sense to me), you meant label, instead of entity ?